.github/CODEOWNERS

```diff
@@ -250,6 +250,7 @@
 /core/templates/dev/head/pages/question_editor/ @aks681
 /core/templates/dev/head/pages/questions_list/ @aks681
 /core/templates/dev/head/pages/question_player/ @vinitamurthi
+/core/templates/dev/head/components/score_ring/ @vinitamurthi
```

Comment:
Add this in sorted order [Line 248]. (This makes it easier to look for a file/dir)

Reply:
Done